docker/.env

```diff
@@ -0,0 +1,7 @@
+FACTORIO_VERSION=latest
```

Comment:
I would move this into the docker-compose files.

Reply:
Done